### DISCUSSION TITLE
Searchable selects: search in the middle of the strings

### DISCUSSION THREAD

**Discussion by lucasromanojf** [livewire/flux#136] [Feature Requests] [answered]
Hello,

currently the searchable select searches only in the beginning of the strings. It would be great if it could search in the middle of them.

**Comment by wevertondcl** [comment]
Yes, I agree! Additionally, some words in Portuguese (Pt-BR) have accents, like names such as João and Márcia. If you don't type the accent, the search won't return results

**Comment by lucasromanojf** [comment]
> Yes, I agree! Additionally, some words in Portuguese (Pt-BR) have accents, like names such as João and Márcia. If you don't type the accent, the search won't return results


Same here, I am from Brazil too 😁

**Comment by LasseRafn** [comment]
Relates to #14 btw

**Comment by joshhanley** [comment]
Thanks for reporting! We have converted this issue to a feature request for now, so we can focus on fixing any bugs first.

But as this is the same request as discussion #126, I'm going to close this one and we can continue the discussion there.

**Comment by ProgrammerZ** [comment]
I would also include ability to search by additional terms for example: `United States` may have additional search terms `['USA', 'America']`. It's a good UX.

**Comment by patrykszady** [comment]
I believe this was added in .19 ? It seems to work now on my end. Can anyone confirm and close this?

**Reply by lucasromanojf** [reply, marked as answer]
Yes, changelog says it was added. Closing now.
